docs/user/openstack/customization.md

```diff
@@ -90,6 +91,37 @@ pullSecret: '{"auths": ...}'
 sshKey: ssh-ed25519 AAAA...
 ```
 
+## Image Overrides
+
+When doing a disconnected installation, the OpenStack platform has the additional requirement that we have locations to download the RHCOS images. The installer downloads these from a location described in
```

Comment:
> When doing a disconnected installation

strictly speaking, that requirement is always present, and not only for disconnected installs.

What about something like:
```
The installer downloads the RHCOS image from a predetermined location (cf [data/data/rhcos.json](/data/data/rhcos.json)). The download URL can be overridden, notably for disconnected installations.
```

Reply:
done